Core/Core/Common/CommonUI/Accessibility/UIAccessibilityAnnouncement.swift

```diff
@@ -16,38 +16,44 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 //
 
+import Combine
+
+public protocol AccessibilityHandler {
+    func post(notification: UIAccessibility.Notification, argument: Any?)
+    var isVoiceOverRunning: Bool { get }
+}
+
+public struct DefaultAccessibilityHandler: AccessibilityHandler {
```

Comment:
no need to change unless you push something anyway
```suggestion
public struct DefaultAccessibilityNotificationHandler: AccessibilityNotificationHandler {
```